gulpfile.js

```diff
@@ -94,10 +124,42 @@ gulp.task('build', function(cb) {
 			gulp.src(paths.src)
 				.pipe(gulp.dest(path.join(getBuildDir(browser), paths.dest)));
 		});
+
+		if (config[browser].manifest) {
+			del(path.join(getBuildDir(browser), config[browser].manifest));
```

Comment:
That's strange, it seems to work fine for me without it :/
Although I have encountered this before; I think I did something weird to fix it (like appending `./` to the dest path)